tvdb_api.py

```diff
@@ -557,6 +557,8 @@ def _loadUrl(self, url, recache = False, language=None):
         if not IS_PY2:
             # Python 3: return content at URL as bytes
             resp = self.session.get(url)
+            if str(resp) == "<Response [404]>":
```

Comment:
Should check `resp.status_code` instead of comparing the str representation of the response. Also might as well check if the response is `>= 400` to catch server errors and so on